ResourceManager/Strings.cs

```diff
@@ -94,26 +86,32 @@ public static string GetLoadingData()
             return Instance._loadingData.Text;
         }
 
+        /// <summary>"branches" translation.</summary>
+        public static readonly TranslationString branches = new TranslationString("Branches");
+        /// <summary>"remotes" translation.</summary>
+        public static readonly TranslationString remotes = new TranslationString("Remotes");
+        /// <summary>"tags" translation.</summary>
+        public static readonly TranslationString tags = new TranslationString("Tags");
+
         public static string GetUninterestingDiffOmitted()
         {
             return Instance._uninterestingDiffOmitted.Text;
         }
 
-        private readonly TranslationString _dateText       = new TranslationString("Date");
-        private readonly TranslationString _authorText     = new TranslationString("Author");
-        private readonly TranslationString _authorDateText = new TranslationString("Author date");
-        private readonly TranslationString _committerText  = new TranslationString("Committer");
-        private readonly TranslationString _commitDateText = new TranslationString("Commit date");
-        private readonly TranslationString _commitHashText = new TranslationString("Commit hash");
-        private readonly TranslationString _messageText    = new TranslationString("Message");
-        private readonly TranslationString _parentsText    = new TranslationString("Parent(s)");
+        private readonly TranslationString _dateText               = new TranslationString("Date");
+        private readonly TranslationString _authorText             = new TranslationString("Author");
+        private readonly TranslationString _authorDateText         = new TranslationString("Author date");
+        private readonly TranslationString _committerText          = new TranslationString("Committer");
+        private readonly TranslationString _commitDateText         = new TranslationString("Commit date");
+        private readonly TranslationString _commitHashText         = new TranslationString("Commit hash");
+        private readonly TranslationString _messageText            = new TranslationString("Message");
+        private readonly TranslationString _parentsText            = new TranslationString("Parent(s)");
```

Comment:
why have these changed?

Reply:
I don't know, it was done by the original author. I can reset it.